lib/src/guild.dart

```diff
@@ -7,5 +7,5 @@ extension GuildExtension on Guild {
   }
 
   /// A URL clients can visit to navigate to this guild.
-  Uri get url => Uri.https(manager.client.apiOptions.host, '/channels/$id');
```

Comment:
Has this been tested? Navigating to `discord.com/channels/guild_id` in the browser does successfully navigate to the guild (yes, the `/channels` is weird).

Reply:
Yeah, it works the same for both `guilds` and `channels` endpoint.
I find less confusing to use guilds instead of channels, but since it's the same behaviour, I can revert this change

Reply:
If /guilds work I would leave it as its less confusing